rllib/core/learner/learner.py

```diff
@@ -272,6 +281,40 @@ def __init__(
         # the final results dict in the `self.compile_update_results()` method.
         self._metrics = defaultdict(dict)
 
+    @OverrideToImplementCustomLogic_CallToSuperRecommended
```

Comment:
Moved here for better ordering of methods (used to be all the way at the bottom of class).